Dockerfile

```diff
@@ -19,6 +19,7 @@ EXPOSE 5090
 
 COPY --from=build /app/out .
 COPY src/Altinn.Notifications.Persistence/Migration ./Migration
+COPY src/Altinn.Notifications/Views ./Views
```

Comment:
I'm not sure how to get this to work through API Management.

Reply:
View is exposed via controller so no changes required